doc/source/user_guide/plotting_with_bokeh.md

```diff
@@ -0,0 +1,139 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(plotting_with_bokeh)=
+
+# Plotting with Bokeh
+
+Arviz has the option to use {doc}`Bokeh <bokeh:index>` as backend which offers interactive plots. Although most of the functions in the {ref}`plot <plot_api>` module work seamlessly with any backend, some advanced plots may require the use of backend specific features. In this guide, advanced plotting with Bokeh will be covered.
+
+This page can be downloaded as a {jupyter-download:script}`Python script <plotting_with_bokeh>`
+or as a {jupyter-download:nb}`Jupyter notebook <plotting_with_bokeh>`.
+
+```{code-cell} ipython3
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+
+az.style.use("arviz-darkgrid")
+
+# Confgure Bokeh as backend
+az.rcParams["plot.backend"] = "bokeh"
+az.output_notebook()
+```
+
+:::{note}
+To display the plots as output in a Jupyter notebook, it's required to call the function  {func}`arviz.output_notebook` beforehand.
+:::
+
+
+## Customizing plots
+
+
+### Using `backend_kwargs`
```

Comment:
```suggestion
(bokeh_backend_kwargs)=
### Using `backend_kwargs`
```

In the plot arguments guide we use `common_arg_name`, so following the same idea I think we should add these anchors here.

Reply:
I added the 6 anchors. Let me know how we should refer to them in the arguments guide. Something like see `ref_matplotlib` or `ref_bokeh`?

Reply:
In the plot arguments guide I'd probably leave two links only, one per backend referencing the top of the file anchor. The argument specific ones should be referenced from the respective descriptions in the docstring (no need to do this here). Once we merge this I think the best would be to update one docstring as an example and then open an issue to signal other people they can also work on this.